lib/aur-sync

```diff
@@ -101,124 +101,70 @@ usage() {
 
 source /usr/share/makepkg/util/util.sh
 source /usr/share/makepkg/util/message.sh
+source /usr/share/makepkg/util/parseopts.sh
 
 if [[ -t 2 && ! -o xtrace ]]; then
     colorize
 fi
 
-if getopt -T || (($? != 4)); then
-    error "$argv0: util-linux getopt required"
-    exit 22
-fi
-
-longopts=allan,bind:,chroot,directory:,force,ignore:,ignorearch,ignore-arch,log
-longopts+=,noconfirm,no-confirm,nover,no-ver,noview,no-view,print,rmdeps,rm-deps
-longopts+=,sign,temp,tar,repo:,database:,root:,upgrades,nhehgvyf,continue,list,git
-longopts+=,makepkg-conf:,pacman-conf:,provides,repo-list,nover-shallow,no-ver-shallow
-shortopts=d:D:AcfglLnprstTu
+opt_short='d:D:AcfglLnprstTu'
+opt_long=('allan' 'bind:' 'chroot' 'directory:' 'force' 'ignore:' 'ignorearch'
+          'ignore-arch' 'log' 'noconfirm' 'no-confirm' 'nover' 'no-ver' 'noview'
+          'no-view' 'print' 'rmdeps' 'rm-deps' 'sign' 'temp' 'tar' 'repo:'
+          'database:' 'root:' 'upgrades' 'nhehgvyf' 'continue' 'list' 'git'
+          'makepkg-conf:' 'pacman-conf:' 'provides' 'repo-list' 'nover-shallow'
+          'no-ver-shallow')
 
-if optstring=$(getopt -o "$shortopts" -l "$longopts" -n "$argv0" -- "$@"); then
-    eval set -- "$optstring"
-else
+if ! parseopts "$opt_short" "${opt_long[@]}" -- "$@"; then
     usage
 fi
-
-tmp=$(mktemp -dt "$argv0".XXXXXXXX)
-tmp_view=$(mktemp -dt view.XXXXXXXX)
-trap 'trap_exit' EXIT
+set -- "${optret[@]}"
```

Comment:
I guess `optret` is the one exception here, in that `parseopts` sets the upper-cased variable `OPTRET` (the other variables are user-defined)

Reply:
Good catch, I was mindless switching the case. I'll fix that real quick.
Thanks.